public/docs/ts/latest/guide/reactive-forms.jade

```diff
@@ -0,0 +1,387 @@
+
+:marked
+  Great! You’ve made a group and you can see that the template and the model are talking to one another by 
+  inspecting the form’s value, but you don’t always need the whole form at once. Sometimes you need to see what’s 
+  happening on one `FormControl`. Time for an aside.
+
+.l-main-section
+a#properties
+:marked
+  ## Inspecting Form Properties
+  A `FormControl` is just a class and if you create an instance, you can inspect its properties. 
+  That means that you can check to see exactly what’s happening with a given `FormControl`. 
+  For example, say you instantiate a new `formControl` (you don’t need to put this in your app):
+
+code-example(format=".").
+  const myControl = new FormControl();
+
+:marked
+  You can easily look at its properties:
+style td, th {vertical-align: top}
+table(width="100%")
+  col(width="10%")
+  col(width="90%")
+  tr
+    th Property
+    th Description
+  tr
+    td <code>myControl.value</code>
+    td
+      :marked
+         Returns the value of a `FormControl`.
+  tr
+    td <code>myControl.status</code>
+    td
+      :marked
+         Returns the status of a `FormControl` as valid, invalid - **KARA, CAN YOU HELP ME HERE?**.
+  tr
+    td <code>myControl.pristine</code>
+    td
+      :marked
+         **KARA, CAN YOU HELP ME HERE, TOO?**.
+  tr
+    td <code>myControl.untouched</code>
+    td
+      :marked
+         **KARA, CAN YOU HELP ME HERE, TOO?**.                                    
+:marked
+  These are just a few examples to give you an idea of what's available. To read more about 
+  **WHERE SHOULD I LINK TO? THE API REF DOESN'T GO INTO THESE METHODS**
+
+  You can inspect an individual `FormControl` in the demo, such as the one for Name, by using 
+  the `.get()` method. If you're curious, add the following to the template after the `form.value | json` 
+  interpolation. It isn't necessary for this demo.
+
++makeExample('reactive-forms/ts/app/hero-signup-4.component.html', 'inspect-value','app/hero-signup.component.html')(format=".")
+
+:marked
+  To get data on a `FormControl` that’s in a `FormGroup`, use the same technique, but with dot notation.
+
++makeExample('reactive-forms/ts/app/hero-signup-4.component.html', 'inspect-child-control','app/hero-signup.component.html')(format=".")
+
+:marked
+  One reason for inspecting the properties of the form is to make sure users fill it out properly. 
+  To do that, the app needs validations. To read more about validating Angular forms, see 
+  [Form Validation](../cookbook/form-validation.html). Ok, back to the demo.
+
+
+.l-main-section
+a#set-data
+:marked
+  ## Setting data with patchValue and setValue
+  You can set default data in the form using `patchValue` or `setValue`. The main difference between 
+  the two is that `patchValue` allows you to provide values as needed, specifying only the fields you 
+  want to provide values for where `setValue` requires that you pass in every control in the form.
+
```

Comment:
The last sentence above is a bit long. Consider breaking it up to make it easier for a reader to digest. Also, do we "pass in every control"? Or "pass a value for every control"?